docs/guide/deployment/coap-gateway.md

```diff
@@ -48,56 +38,233 @@ Follow [OCF Device To Cloud Services Specification](https://openconnectivity.org
+```
+
+## YAML Configuration
+### Logging
+
+| Property | Type | Description | Default |
+| ---------- | -------- | -------------- | ------- |
+| `log.debug` | bool | `Set to true if you would like to see extra information on logs.` | `false` |
+| `log.dumpCoapMessages` | bool | `Set to true if you would like to dump coap messages on logs.` | `false` |
+
+### CoAP API
+CoAP API of Coap Gateway Service as defined in [uri](https://github.com/plgd-dev/cloud/blob/docs-deployment-grpc-gateway/coap-gateway/uri/uri.go) and [swagger](https://raw.githubusercontent.com/openconnectivityfoundation/cloud-services/master/swagger2.0/oic.wk.rd.swagger.json). 
+
+| Property | Type | Description | Default |
+| ---------- | -------- | -------------- | ------- |
+| `api.coap.address` | string | `Listen specification <host>:<port> for coap client connection.` | `"0.0.0.0:9100"` |
+| `api.coap.externalAddress` | string | `External address including public domain/IP for coap client connection.` | `"0.0.0.0:9100"` |
+| `api.coap.maxMessageSize` | int | `Max message size which can be send/receive via coap.` | `262144` |
+| `api.coap.goroutineSocketHeartbeat` | string | `Interval time to check live service.` | `4s` |
+| `api.coap.keepAlive.timeout` | string | `Time limit to close inactive connection.` | `20s` |
+| `api.coap.blockwiseTransfer.enabled` | bool | `If true, enable blockwise transfer of coap message.` | `true` |
+| `api.coap.blockwiseTransfer.blockSize` | int | `Size of blockwise transfer block.` | `1024` |
+| `api.coap.tls.enabled` | bool | `If true, require server certificate for ssl connection.` | `true` |
+| `api.coap.tls.caPool` | string | `File path to the root certificate in PEM format which might contain multiple certificates in a single file.` |  `""` |
+| `api.coap.tls.keyFile` | string | `File path to private key in PEM format.` | `""` |
+| `api.coap.tls.certFile` | string | `File path to certificate in PEM format.` | `""` |
+| `api.coap.tls.clientCertificateRequired` | bool | `If true, require client certificate.` | `true` |
+
+### Event Bus
+Plgd cloud uses NATS messaging system as a event bus.
+
+| Property | Type | Description | Default |
+| ---------- | -------- | -------------- | ------- |
+| `clients.eventBus.goPoolSize` | int | `Number of routines to process events in projection.` | `16` |
+| `clients.eventBus.nats.url` | string | `URL to nats messaging system.` | `"nats://localhost:4222"` |
+| `clients.eventBus.nats.tls.caPool` | string | `root certificate the root certificate in PEM format.` |  `""` |
+| `clients.eventBus.nats.tls.keyFile` | string | `File name of private key in PEM format.` | `""` |
+| `clients.eventBus.nats.tls.certFile` | string | `File name of certificate in PEM format.` | `""` |
+| `clients.eventBus.nats.tls.useSystemCAPool` | bool | `If true, use system certification pool.` | `false` |
+
+### Authorization Server Client
+Authorization server client configuration to connect internally.
```

Comment:
Somewhere you escape it with backshlash, somewhere not. Please unify.

Reply:
@ondrejtomcik 
What do you mean `backslash` ? Give me more detail.

Reply:
Escaping with ‘ character.

Reply:
I don't use backslash character with single quotation mark together on any documents. uhmm...  I'm using goland for editing md files on Mac. Is it has some problem in format compatibility ?

Reply:
It's OK now. Not sure where I saw the difference.